src/content/add-to-app/ios/add-flutter-screen.md

```diff
@@ -465,11 +440,12 @@ For instance:
 import Foundation
 import Flutter
 
-class AppDelegate: UIResponder, UIApplicationDelegate, FlutterAppLifeCycleProvider, ObservableObject {
+@Observable
+class AppDelegate: UIResponder, UIApplicationDelegate, FlutterAppLifeCycleProvider {
 
   private let lifecycleDelegate = FlutterPluginAppLifeCycleDelegate()
 
-  let flutterEngine = FlutterEngine(name: "flutter_nps_engine")
+  let flutterEngine = FlutterEngine(name: "my flutter engine")
```

Comment:
`flutter_nps_engine` was probably from one of the earlier demo project. Update it to be consistent with other part of this doc.